### ISSUE TITLE
I was asked about Open R/O suggestion but 3.9.1 won't launch

### ISSUE THREAD

**Issue by JoeErsatz** [sqlitebrowser/sqlitebrowser#986]
### Details for the issue
Kindly pardon this bizarre interruption. My mind is going fast as I age, though I'm relatively young (66) by popular standards. Really, I have trouble focusing. No, this isn't particularly important since I have minimal use for DB SQLite Browser and my current version 3.8.0 does what I need it to do. I was contacted in January saying that my suggestion for Open in read-only mode was adopted and would I kindly test it. Excuse me but I did not do so. But because I kept getting a message touting a newer version available (yes I know I can stifle that message) I did eventually download your DB.Browser.for.SQLite-3.9.1-win32 and it installed without a balk but when I went to run the installed program I got the system error message that it wasn't "a valid Win32 application". This surprises and confounds me some unless it maybe relates to WXP no longer being supported. Of course I tried downloading anew (some weeks later) but the result was identical. Sorry to bother you kind guys.

#### I'm opening this issue because:

- [ ] DB4S is crashing
- [ ] DB4S has a bug
- [ ] DB4S needs a feature
- [ X] DB4S has another problem

#### I'm using DB4S on:

- [ X] Windows: ( _version:_ _WXP__ )
- [ ] Linux: ( _distro:_ ___ )
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### I'm using DB4S version:

- [ X] 3.9.1
- [ ] 3.9.0
- [ ] Other: ___

#### I have also:

- [no ] Tried out the nightly version: https://github.com/sqlitebrowser/sqlitebrowser#windows
- [ ] Searched for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by chrisjlocke** [comment]
Indeed, the newer version of DB4S doesn't support XP.  It is possible, but due to the methods of building the application we use, not easily 'automated'.
As a side note, Windows XP is quite an old operating system now and is unsupported as far as security patches, etc, goes, so if you use it in a 'production' environment (or for more than just 'testing') then it shouldn't be used to go online.
As another side note, more and more programs will drop support for Windows XP, so its not just DB4S which will fail to run.
As another side note, its still free to upgrade to Windows 10.  Not that really helps the above issue, and of course, comes with its own problems...

**Comment by JoeErsatz** [comment]
Many thanks for that. Sure, I would ditch this machine in a flash and get a W10 PC (this one, I'm fairly sure, isn't W10 hardware compatible). Another PC of mine runs W10 and I'm disappointed I can't get the game Snood 4.10 to operate right on it. But your advice hasn't fallen on deaf ears so just know that - thanks again. Over and out.

**Comment by JoeErsatz** [comment]
Thank you for not forgetting about me, though I was content using v3.8. But YES, I was able to make the cited patch (using an old pgm I had called binedit) and now my v3.9.1 works fine under WXP. I don't immediately see where/how one opens a database read-only, but I trust it is there somewhere/somehow, and don't think for a moment that this is any longer the teensiest bit important to me anymore. Thank you nonetheless!!
